packages/components/src/components/toolbar/constants.ts

```diff
@@ -0,0 +1 @@
+export const TOOLBAR_ITEM_TAG_NAME = 'kol-toolbar-item';
```

Comment:
Das Auslagern sollten wir erst dann machen, wenn die Konstante in mehreren anderen Dateien verwendet wird.